### PR TITLE
move-prometheus-boshreleas-to-cloudfoundry

### PR DESCRIPTION
**Pull request by saadia-kadiri**
prometheus-boshrelease has moved  from bosh-prometheus gh org to cloudfoundry gh org

### CODE REVIEW

File: index.yml [hunk 1]
```diff
@@ -318,7 +318,7 @@
 - url: https://github.com/cloudfoundry-community/sawmill-boshrelease
 - url: https://github.com/SAP/ipsec-release
   min_version: 3
-- url: https://github.com/bosh-prometheus/prometheus-boshrelease
+- url: https://github.com/cloudfoundry/prometheus-boshrelease
 - url: https://github.com/bosh-prometheus/node-exporter-boshrelease
```
Comment (bgandon) [inline index.yml:322]: This release has also moved and is subject to being updated here either.
See: https://github.com/cloudfoundry/community/blob/a81540a/toc/working-groups/foundational-infrastructure.md?plain=1#L385-L389

**Review by bgandon — CHANGES_REQUESTED** [review]

**Comment by jpalermo** [comment]
Before merging @saadia-kadiri, I just want to confirm expected behavior from this change.

The pipelines will be updated such new releases will not be generated and added [here](https://github.com/cloudfoundry/bosh-io-releases-index/tree/main/github.com/bosh-prometheus/prometheus-boshrelease)

The existing releases will continue to show up [here](https://bosh.io/releases/github.com/bosh-prometheus/prometheus-boshrelease?all=1) on bosh.io, but you will no longer see `bosh-prometheus/prometheus-boshrelease` listed on the [index page](https://bosh.io/releases/) of releases.

All existing releases, and all future releases will be built for `cloudfoundry/prometheus-boshrelease` and that will show up on the releases index page.

So the main downside would be if somebody is consuming the release from the existing path, they will not see any future releases there and at some point they would hopefully realize this and switch to the new path.

File: index.yml (hunk 1 above)
Comment (saadia-kadiri) [reply index.yml]: Hello @bgandon, cloudfoundry/bosh_exporter, cloudfoundry/cf_export and cloudfoundry/firehose_exporter are not in https://github.com/cloudfoundry/bosh-io-releases. I have modified the path of[ node-exporter-boshrelease](https://github.com/cloudfoundry/node-exporter-boshrelease)

Comment (saadia-kadiri) [reply index.yml]: Hello [@bgandon](https://github.com/bgandon) , **cloudfoundry/bosh_exporter**, **cloudfoundry/cf_exporter** and **cloudfoundry/firehose_exporter** are not in https://github.com/cloudfoundry/bosh-io-releases. I also modified the path of [node-exporter-boshrelease](https://github.com/cloudfoundry/node-exporter-boshrelease) in this PR

**Comment by jpalermo** [comment]
@saadia-kadiri , this seems fine as long as you're all fine with the behavior I outlined above?

**Comment by gmllt** [comment]
@jpalermo, So if @saadia-kadiri modifies the PR to only add the new repositories without removing the old ones, someone who wanted to see the old release could still see it.

Is there a way to indicate that the old path is deprecated in favor of the new path?

**Comment by jpalermo** [comment]
Keeping both would allow someone who didn't know about the change to continue to get new updates. Both would show up on the [releases page](https://bosh.io/releases/), which might cause some confusion.

There is not currently any way to annotate a release as deprecated.

An ideal solution would be to add sort of `aliases` key to this repo so any previous release paths could be handled by the new path. You'd need to change the [web app](https://github.com/cloudfoundry/bosh-io-web) to handle that new structure though. I haven't looked at the code closely, but I kinda bet the change would be pretty straight forward.

**Review by rkoster — APPROVED** [review]

**Comment by rkoster** [comment]
Thanks! @saadia-kadiri

**Comment by rkoster** [comment]
Pipeline changes have been synced:
- https://bosh.ci.cloudfoundry.org/teams/x-bosh-io/pipelines/releases-cloudfoundry/jobs/prometheus-boshrelease/builds/1
- https://bosh.ci.cloudfoundry.org/teams/x-bosh-io/pipelines/releases-cloudfoundry/jobs/node-exporter-boshrelease/builds/1

**Pull request merged**